### PR TITLE
update share url

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Updates ShareBoards Url, since `boards` requires authentication. Shared Boards may be viewed by unauthorized users.

#### Ticket Link
  Fixes https://github.com/mattermost/focalboard/issues/1046

### CODE REVIEW

File: webapp/src/components/shareBoardComponent.tsx [hunk 1]
```diff
@@ -76,8 +76,7 @@ const ShareBoardComponent = React.memo((props: Props): JSX.Element => {
     shareUrl.searchParams.set('r', readToken)
 
     if (match.params.workspaceId) {
-        const newPath = generatePath(':basename/workspace/:workspaceId/shared/:boardId/:viewId', {
-            basename: Utils.getFrontendBaseURL(),
+        const newPath = generatePath('plugins/focalboard/workspace/:workspaceId/shared/:boardId/:viewId', {
```
Comment (chenilim) [inline webapp/src/components/shareBoardComponent.tsx:79]: I'm not 100% sure, but do we need a / at the beginning? Also, might be a separate issue, but will this work if the base Mattermost path is not on the root?

**Review by chenilim — APPROVED** [review]
Had a comment about the URL, but LGTM if Jesus is ok.

File: webapp/src/components/shareBoardComponent.tsx (hunk 1 above)
Comment (jespino) [reply webapp/src/components/shareBoardComponent.tsx:79]: Yes, I think we need the `/` at the beginning, but I don't know if the `shareurl.pathname = newPath` is already doing it.

**Review by jespino — APPROVED** [review]
LGTM

File: webapp/src/components/shareBoardComponent.tsx (hunk 1 above)
Comment (sbishel) [reply webapp/src/components/shareBoardComponent.tsx]: Nope...updated.

**Pull request merged**